### ISSUE TITLE
Plot the underlying generative function

### ISSUE THREAD

**Issue by CL-BZH** [mml-book/mml-book.github.io#452]
**Suggestion**
In figures 9.4 and 9.5 it would be nice to have the underlying generative function
f(x) = − sin(x/5) + cos(x).
Especially in figure 9.5 this would show the problem of over-fitting where we have a perfect match for training data but we are "far" from the generative function.

**Comment by mpd37** [comment]
Yes and no. In this case, we do know the ground truth, but in reality we don't. And we need to be able to talk about overfitting in absence of ground truth.

**Comment by CL-BZH** [comment]
I agree.
However, there is always an underlying generative function (of course we don't know it otherwise we would not have to do inference). I think that showing it in figure 9.5 would let one visualize why overfitting is bad.
Thanks,
Chris.
